gazebo_plugins/include/gazebo_plugins/gazebo_ros_camera.hpp

```diff
@@ -70,6 +70,8 @@ class GazeboRosCameraPrivate;
       <!-- Projection matrix translation Tx, Ty between stereo cameras-->
       <Tx>240.5</Tx>
       <Ty>0</Ty>
+      <!-- Full 3x3 rectification matrix -->
```

Comment:
It might be worth documenting here that the values are in row-major order.

Reply:
Thanks, added that to the comment.